src/asynchronous/server.rs

```diff
@@ -292,6 +292,7 @@ async fn handle_request(
         return;
     }
 
+    let stream_id = header.stream_id;
```

Comment:
Why define a new stream_id, why not use header.stream_id directly just as the others did?

Reply:
header.stream_id would be moved when used in line 332

Reply:
Please defined it where it needed, don't defined it in the outer scope.

Reply:
done